pkg/render/node_test.go

```diff
@@ -993,8 +1014,11 @@ var _ = Describe("Node rendering tests", func() {
       "datastore_type": "kubernetes",
       "mtu": 0,
       "nodename_file_optional": false,
-      "log_level": "Info",
+      "log_level": "Debug",
       "log_file_path": "/var/log/calico/cni/cni.log",
+      "log_file_max_size": 1,
+	  "log_file_max_age": 5,
```

Comment:
weird indentation on this one

Reply:
ah! looks like this is a tab not a space! I'll fix that.